colossalai/inference/modeling/models/nopadding_llama.py

```diff
@@ -436,6 +481,5 @@ def forward(self, hidden_states: torch.Tensor) -> torch.Tensor:
         """
         hidden_states = hidden_states.expand(2, -1, -1)
         gate_up_proj_out = torch.bmm(hidden_states, self.gate_up_weight)
-        act_out = torch.nn.functional.silu(gate_up_proj_out[0], inplace=True)
-        tmp_out = act_out * gate_up_proj_out[1]
-        return torch.mm(tmp_out, self.down_proj_weight)
+        act_out = inference_ops.silu_and_mul(gate_up_proj_out)
```

Comment:
Where does `use_cuda_kernel` go

Reply:
The silu_and_mul doesn't has triton kernel, so we may not need use_cuda_kernel.